### DISCUSSION TITLE
Linq2Db EF 7 PostgreSQL timestamp behavior

### DISCUSSION THREAD

**Discussion by datvladik** [linq2db/linq2db#3888] [Q&A]
Hello,

On my project we are using EF 7 with Npgsql provider (https://www.npgsql.org/efcore/release-notes/7.0.html) and linq2db.EntityFrameworkCore 7.1.0.
 
Since release 6.0 npgsql driver store DateTime as timestamp with time zone (i know i can turn off this behavior) https://www.npgsql.org/efcore/release-notes/6.0.html?tabs=annotations

But when i try use linq2db like this

` await _context.Contents.Set(x => x.ModifiedAt, DateTime.UtcNow).UpdateAsync()`
 
 I get an error  "Cannot write DateTime with Kind=Unspecified to PostgreSQL type
'timestamp with time zone', only UTC is supported" 

Maybe I'm doing something wrong?

**Comment by MaceWindu** [comment]
Closing in favor of #3895
